### PR TITLE
updating the CLI to use the latest version of webhost, java worker and python worker

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: src/Azure.Functions.Cli/Helpers/PythonHelpers.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace Azure.Functions.Cli.Helpers
 {
     public static class PythonHelpers
     {
-        private static readonly string[] _workerPackages = new[] { "azure-functions==1.0.0a4", "azure-functions-worker==1.0.0a4" };
+        private static readonly string[] _workerPackages = new[] { "azure-functions==1.0.0a5", "azure-functions-worker==1.0.0a6" };
```
Comment (ahmelsayed) [inline src/Azure.Functions.Cli/Helpers/PythonHelpers.cs]: is `azure-functions` intentionally `0a5` not `0a6`?

Comment (soninaren) [reply src/Azure.Functions.Cli/Helpers/PythonHelpers.cs]: @pragnagopa can you comment on this?

Comment (pragnagopa) [reply src/Azure.Functions.Cli/Helpers/PythonHelpers.cs]: Yes. No changes in azure-functions from a5  but azure-functions-worker had few bug fixes then went in.

**Comment by pragnagopa** [comment]
> any changes to other workers? node, powershell?
cc: @mhoeger 
No changes for Node. 

@tylerl0706 / @daxian-dbw - Any updates for the powershell worker you need to release with the CLI?

**Comment by daxian-dbw** [comment]
@pragnagopa Yes, we would like to release a newer version, including some bug fixes.

**Comment by soninaren** [comment]
@daxian-dbw is that newer version ready to be consumed? If not we can always take it as a part of the next release

**Comment by daxian-dbw** [comment]
@soninaren No, it's not in MyGet yet, I will build a package and ask @pragnagopa to push to MyGet now.

**Comment by soninaren** [comment]
@daxian-dbw , Perfect, i will wait for the push to MyGet and then will update my PR to include the powershell worker update.

**Comment by pragnagopa** [comment]
@soninaren - please update powershell worker to Microsoft.Azure.Functions.PowerShellWorker.0.1.20-alpha

**Comment by soninaren** [comment]
> @soninaren - please update powershell worker to Microsoft.Azure.Functions.PowerShellWorker.0.1.20-alpha

done

**Pull request merged**